defnet/tcp_server.lua

```diff
@@ -83,10 +82,14 @@ function M.create(port, on_data, on_client_connected, on_client_disconnected)
 	function server.start()
 		print("Starting TCP server on port " .. port)
 		local ok, err = pcall(function()
-			local skt, err = socket.bind("*", port)
+			local skt, err = socket.tcp()
```

Comment:
I'm not sure about this to be honest. The "socket/socket.lua" is from the LuaSocket library and contains helpers to set up socket connections. It is actually the same file as in builtins/scripts/socket.lua. And both come from the official LuaSocket GitHub repo:

https://github.com/lunarmodules/luasocket/blob/master/src/socket.lua#L27-L54

As you can see from the code it is a helper to set up either an IPV4 or IPV6 server socket. You changes does a lot of the same but misses the IPV6 bit.

Reply:
Absolutely! Go for it!